docs/CHANGELOG.asciidoc

```diff
@@ -38,15 +38,484 @@ Review the following information about the {kib} 8.3.0 release.
+** v3_windows_anomalous_process_creation
+** v3_windows_anomalous_script
+** v3_windows_anomalous_service
+** v3_windows_anomalous_user_name_ecs
+** v3_windows_rare_metadata_process
+** v3_windows_rare_metadata_user
+** v3_windows_rare_user_runas_event
+** v3_windows_rare_user_type10_remote_login
+====
+      
+[discrete]
+[[deprecation-130336]]
+.Updates the default legend size
+[%collapsible]
+====
+*Details* +
+In the *Lens* visualization editor, the *Auto* default for *Legend width* has been deprecated. For more information, check {kibana-pull}130336[#130336].
+
+*Impact* +
+When you create *Lens* visualization, the default for the *Legend width* is now *Medium*. 
+====
+      
+[discrete]
+[[deprecation-122075]]
+.Deprecates `xpack.data_enhanced.*`
+[%collapsible]
+====
+*Details* +
+In kibana.yml, the `xpack.data_enhanced.*` setting is deprecated. For more information, check {kibana-pull}122075[#122075].
+
+*Impact* +
+Use the `data.*` configuration parameters instead.
+====
+      
+[float]
+[[features-8.3.0]]
+=== Features
+
+{kib} 8.3.0 adds the following new and notable features.
+
+Alerting::
+* Adds circuit breaker for max number of actions by connector type {kibana-pull}128319[#128319]
+* Adds `bulkEdit` method to alerting rulesClient and internal _bulk_edit API, that allow bulk editing of rules {kibana-pull}126904[#126904]
+
+Cases::
+* Adds average time to close metric in Cases {kibana-pull}131909[#131909]
+* View all alerts attached to a case in the alerts table. The feature is experimental {kibana-pull}131883[#131883]
+* Adds severity field to Cases {kibana-pull}131626[#131626]
+* Adds the ability to delete comments in Cases {kibana-pull}130254[#130254]
+
+Dashboard::
+Enables the new controls by default {kibana-pull}131341[#131341]
+
+Discover::
+* Adds the ability to edit dataView, query, & filters {kibana-pull}131688[#131688]
+* Extend Elasticsearch query rule with search source based data fetching {kibana-pull}124534[#124534]
+
+Elastic Security::
+For the Elastic Security 8.3.0 release information, refer to {security-guide}/release-notes.html[_Elastic Security Solution Release Notes_].
+
+Fleet::
+Changes to agent upgrade modal to allow for rolling upgrades {kibana-pull}132421[#132421]
+
+Lens & Visualizations::
+* Adds method to re-link visualizations with missing `SavedSearch` {kibana-pull}132729[#132729]
+* Adds support of Data View switching for Agg-Based visualizations {kibana-pull}132184[#132184]
+* Adds `showTooltip` argument {kibana-pull}130530[#130530]
+* Adds `orderBucketsBySum` {kibana-pull}130457[#130457]
+* Adds lines visibility for lines chart {kibana-pull}130452[#130452]
+* Adds `pointsRadius`, `showPoints`, and `lineWidth` {kibana-pull}130391[#130391]
+* Adds mark size configuration {kibana-pull}130361[#130361]
+* Adds `xyVis` split chart {kibana-pull}130229[#130229]
+
+Machine Learning::
+* Adds ML expected model bounds as an option to Comparison controls {kibana-pull}132456[#132456]
```

Comment:
This one should be moved into the Observability section as it is in the APM section to Kibana.